etc/packaging/appimages/cartographer-module-aarch64.yml

```diff
@@ -26,58 +26,19 @@ AppDir:
     - arm64
     allow_unauthenticated: true
     sources:
-    - sourceline: deb [trusted=yes] http://deb.debian.org/debian bullseye main
-    - sourceline: deb [trusted=yes] http://deb.debian.org/debian-security bullseye-security main
-    - sourceline: deb [trusted=yes] http://deb.debian.org/debian bullseye-updates main
-    - sourceline: deb [trusted=yes] https://us-apt.pkg.dev/projects/static-file-server-310021 bullseye main
+    - sourceline: deb [trusted=yes] http://deb.debian.org/debian bookworm main
```

Comment:
[note] Upgrading to bookworm to use non-dev packages which are smaller and unavailable in bullseye. This upgrade also matches the new red-env used in workflows. Appimage will be backward compatible to bullseye